### ISSUE TITLE
tailscale up --force-reauth does not require --accept-risk=lose-ssh

### ISSUE THREAD

**Issue by andrew-d** [tailscale/tailscale#6377]
### What is the issue?

I received a report from a friend that "`--accept-risk=lose-ssh` does not kick in if you do `--force-reauth`, and then you lose connectivity to the host".

We should handle this case.

**Comment by Mansarde** [comment]
Your title seems to imply something different than what the quote of your friend says.

If `--accept-risk=lose-ssh` has been set, then I'd expect `--force-reauth` to go through and make me lose ssh connection, because I explicitly accepted the risk.
Only if I _didn't_ set `--accept-risk=lose-ssh` would I expect `--force-reauth` to not go through.

Just for clarity, I'm assuming you mean the latter case (i.e. what your title describes), right?

**Comment by zofrex** [comment]
I tested this to confirm what the current behaviour is:

### Steps to reproduce

I SSH'd from one machine to another via Tailscale SSH and issued the command:

```$ tailscale up --ssh --force-reauth```

### Expected behaviour

A warning that this command would disconnect my SSH session, similar to what `tailscale down` does for example:

```$ tailscale down
You are connected over Tailscale; this action will disable Tailscale and result in your session disconnecting.
To skip this warning, use --accept-risk=lose-ssh
Continuing in 5 seconds...
```

I expect a warning like this to be shown when doing `up --force-reauth` unless `--accept-risk=lose-ssh` is also passed.

### Actual behaviour

The machine immediately dropped off the Tailnet without any warning, and my SSH session hung, without receiving any output from the tailscale CLI (and without receiving the URL required to reauth). This happens regardless of whether `--accept-risk=lose-ssh` is passed or not.

**Comment by zofrex** [comment]
This is now released with 1.40.0
